Bookish.Web/Controllers/HomeController.cs

```diff
@@ -65,12 +69,24 @@ public IActionResult NewBookPost(string title, string author, string isbn, int t
             }
 
             bookishService.AddBook(title, author, isbn, totalCopies);
-            return RedirectToAction("Barcodes");
+            return RedirectToAction("Barcodes", new { isbn });
         }
 
-        public IActionResult Barcodes()
+        [Route("/Home/Barcodes/{isbn}")]
+        public IActionResult Barcodes(string isbn)
         {
-            return View();
+            var bookCopies = bookishService.GetCopies(isbn).ToList();
+            var title = bookCopies.First().Title;
+            var author= bookCopies.First().Author;
```

Comment:
I'd put this logic in the view controller. And remember whitespace around operators such as `=` 🙂